### DISCUSSION TITLE
Allow reordering of templates inside a template group

### DISCUSSION THREAD

**Discussion by liljefelt** [ExpressionEngine/ExpressionEngine#3133] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Most of the time when I'm working on a new website, the navigation is not listed in alphabetical order.
I usually make a mental map between the site navigation and template structure and having to scan the list of templates in the CP for a template that I mentally have placed at the top of the list because it's the first nav item, is a bit cumbersome.

**Describe the solution you'd like**
I would like to be able to reorder templates inside a template group.
It would work similarly to how reordering of template groups works today, drag and drop.

**Comment by intoeetive** [comment]
I like the idea and I think it might be not too hard to implement.

@TomJaeger FYI

**Reply by liljefelt** [reply]
Glad to hear! Fingers crossed :)
